<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranWordReview</class>
 <widget class="QDialog" name="TranWordReview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>测试结束！</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(77, 83, 49);</string>
  </property>
  <widget class="QLabel" name="labelRight">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>73</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>正确：28</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>23</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>测试结束！#3#@~!&amp;^$%??!1#&gt;&gt;&amp;(*80</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWrong">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>133</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>错误：3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonYes">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>248</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>是</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelReview">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>248</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>是否进入复习模式？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>248</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(117, 133, 67);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>否</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>93</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;#$%^&amp;amp;*()_)&amp;amp;^%$#55&amp;amp;^4^&amp;amp;#56789)(^dfdfd%5(&amp;gt;&amp;lt;?^$@#$095(B$7(*%B&amp;amp;%$%*$567)%$567&amp;gt;&amp;lt;$5*(。&amp;lt;56(*6678o54tyuio[/]p;Y&amp;amp;*^^90#$%^&amp;amp;*(98&amp;amp;^%$#@#$%^&amp;amp;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>193</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>总计：31</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
